src/App.js

```diff
@@ -33,14 +40,16 @@ const App = () => {
   const createDeck = () => {
     console.log("createDeck invoked")
   }
+
   return (
     <>
       <Header currentUser={currentUser} logout={logout}/>
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/decknew" element={<DeckNew createDeck={createDeck} currentUser={currentUser}/>} />
         <Route path="/decks" element={<DeckIndex decks={decks}/>} />
-        <Route path="/decks/:deckId" element={<CardIndex cards={cards}/>} />
+        <Route path="/decks/:deckId/:cardId" element={<CardIndex cards={cards} />} />
```

Comment:
There will be no need for cardID as a param since this is the CardIndex component route which should display all cards.